recipe/meta.yaml

```diff
@@ -1,32 +1,34 @@
 package:
     name: esmpy
-    version: 7.0.0
+    version: 7.1.0r
 
 source:
-    git_url: git://git.code.sf.net/p/esmf/esmf
-    git_tag: ESMF_7_0_0
+    url: https://sourceforge.net/code-snapshots/git/e/es/esmf/esmf.git/esmf-esmf-e3ac95b2e25a36d6cbfa68ed93eb1645fea29fcd.zip
+    sha256: 8d3b77b552705901542c2bc6c24bddfdd8daa8c01a8b790885b93392e3c627c6
 
 build:
-    number: 1
-    skip: True  # [win or py3k]
+    number: 0
+    skip: True  # [win]
 
 requirements:
     build:
         - python
         - numpy
-        - esmf
+        - esmf ==7.1.0r
```

Comment:
We should do `esmf {{ version }}` here and below to avoid errors when updating the package.

Reply:
👍 Pushed an update. Thanks for the review!